app/assets/javascripts/cookiesalert/cookies_alert.js

```diff
@@ -9,30 +9,33 @@ var cookies = {
       cookies.btn = obj.btn || cookies.btn
       cookies.expires = obj.expires || cookies.expires
     }
- 
-    if (Cookies.get('cookies_alert') === undefined) {
+
+    if (Cookies.get('cookies_consent') === undefined) {
       $(cookies.div).css({ visibility : 'visible' })
       $(cookies.btn).click(function () {
         cookies.accept();
+        Cookies.set('cookies_consent', 'true');
```

Comment:
this is already being done in `accept` which is called in L16

Reply:
Removed